aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -119,6 +119,10 @@ module aptos_framework::reconfiguration {
             return
         };
 
+        // If `RECONFIGURE_WITH_DKG` is enabled,
+        // `reconfiguration_with_dkg::start()` already marked it correctly, and this invocation becomes a no-op.
+        reconfiguration_state::try_mark_as_in_progress();
```

Comment:
When would this be called more than once?

Reply:
As described in the comments above: when `RECONFIGURE_WITH_DKG` is enabled, `reconfiguration_with_dkg::start()` will call it and this invocation is no-op.

But when `RECONFIGURE_WITH_DKG` is disabled and we do default reconfig, this call does the work to record start time.